src/main.h

```diff
@@ -51,6 +51,11 @@ inline bool MoneyRange(int64_t nValue) { return (nValue >= 0 && nValue <= MAX_MO
 /** Threshold for nLockTime: below this value it is interpreted as block number, otherwise as UNIX timestamp. */
 static const unsigned int LOCKTIME_THRESHOLD = 500000000; // Tue Nov  5 00:53:20 1985 UTC
 
+// returns percentage reward per year
+inline int64_t GetCoinYearReward() {
```

Comment:
once we get the RoI changes in, we need to rewrite this function with that one 👍

Reply:
GetCoinYearReward is included in the stakesplitthreshold branch/pr. Once I figure out why the commit isn't going in it should be available for review.